.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+    "env": {
+        "test": {

```

Comment:
why do we need this preset configuration?

Reply:
To be version-agnostic to the node version that's running the tests.

Reply:
OK we'll move that into the uxf_webpack instead